cli/bin/cherry.js

```diff
@@ -228,7 +229,7 @@ program
         const files = await getFiles()
         const codeOwners = new Codeowners()
         const occurrences = await findOccurrences({ configuration, files, codeOwners })
-        await upload(apiKey, configuration.project_name, committedAt, occurrences)
+        await upload(apiKey, configuration.project_name, committedAt, occurrences, configuration.metrics)
```

Comment:
I think you also need to do something with option.metric.

Reply:
We spoke about all the metrics not for only one? the option.metric is for only one?

Reply:
I did a proposition for the run case

Reply:
From what I recall, configuration.metrics does not include metrics coming from plugins. 
So this would only fix the issue for a part of the metrics. 
I'll take a look at it now. See what we can do... 🤔